src/ontology/mondo-ingest.Makefile

```diff
@@ -421,14 +448,29 @@ matches: lexical-matches extract-unmapped-matches
 #  - icd10cm/icd10who ttl -> owl: https://github.com/monarch-initiative/mondo-ingest/issues/138
 #  - No rule to make target 'mirror/ONTOLOGY.owl': https://github.com/monarch-initiative/mondo-ingest/issues/137
 # Maybe solved now by using $(COMPONENTSDIR)/
+# TODO: Figure out why -P is no longer working when using this in ODK, and add it back in.
+#  Error: No such option: -P
+#  RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
 $(COMPONENTSDIR)/%.db: $(COMPONENTSDIR)/%.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
-	@rm -f $(COMPONENTSDIR)/%-relation-graph.tsv.gz
-	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	@rm -f $(COMPONENTSDIR)/$*-relation-graph.tsv.gz
+	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv || RUST_BACKTRACE=full semsql make $@
```

Comment:
@matentzn @hrshdhgd My apologies, but I was not able to figure out why I am having this error now:
> `Error: No such option: -P`
Some months ago, I believe that Harshad added `-P config/prefixes.csv` after it was added to `semsql`. And I remember that when I first ran this goal after that, I got this error. And I don't think I've gotten it since.

However now, whether I am running the latest ODK (1.4) or the dev ODK, I am still getting this error:
```
(mondo-ingest) sh run.sh semsql -P
Running  with -Xmx20G of memory for ROBOT and Java-based pipeline steps.
Usage: semsql [OPTIONS] COMMAND [ARGS]...
Try 'semsql --help' for help.

Error: No such option: -P
```

I checked `pip freeze`, and these containers show that `semsql` is on `semsql==0.2.5`. I also installed the latest locally `semsql==0.3.2`, and I'm also getting the same error.

I uninstalled `semsql` from my `mondo-ingest` virtual environment, as well as globally, and I'm still getting the same error.

Do you guys get the same error? If not, any idea why this is happening to me?

Reply:
Did you run `docker pull obolibrary/odkfull:dev`?

Reply:
(this is a recent feature in semsql but I had added it to the ODK dev image.

Reply:
Yeah, my docker `dev` image and `latest` / `v1.4` images are all up-to-date. I just checked. I'm baffled, because I'm pretty sure that the `-P` flag was working for me just last week.

Reply:
I made a new `dev` release, can you try that?

Reply:
@matentzn I just puled the latest `obolibrary/odkfull:dev` tried it, but it's not working. For some reason, it's still on `semsql==0.2.5`, which was released in December. 

```
ODK_TAG=dev sh run.sh pip freeze | grep semsql
semsql==0.2.5
```

I just checked the `git log` for SemanticSQL and I see this feature was added mid March. We want `semsql==0.3.2`, the latest version, which was [released](https://github.com/INCATools/semantic-sql/releases) right after that.

Reply:
Cant be Right: https://github.com/INCATools/ontology-development-kit/blob/master/constraints.txt#L245

semsql should be 0.3.2. Please triple check.. I am losing my mind over this issue slowly slowly.

Reply:
MKe sure that run.sh.conf does not overwrite the environment variable

Reply:
Yeah I just did that:

```
docker pull obolibrary/odkfull:dev
sh run.sh bash
root@bb43e0afbcee:/work/src/ontology# pip list | grep sems
semsql                        0.3.2
root@bb43e0afbcee:/work/src/ontology# 
```

I think your problem is the content of `run.sh.conf`. Change that to `dev`!